client/components/domains/contact-details-form-fields/managed-contact-details-form-fields.jsx

```diff
@@ -243,7 +241,7 @@ export class ManagedContactDetailsFormFields extends Component {
 		return (
 			<div className="contact-details-form-fields__row">
 				<Input
-					label={ this.props.translate( 'Alternate Email Address' ) }
+					label={ this.props.translate( 'GSuite Contact Email Address' ) }
```

Comment:
What's the reason for changing that label? This seems slightly redundant with the `G Suite Account Information` heading. `GSuite` should be written `G Suite` by the way 🙂.

Reply:
A good question. I figured that it sounded strange to have "Alternate email address" as a label when there's no "Email address" field. Since this is only used for GSuite (I think?) I figured we might as well make it clear what the field is for.

> GSuite should be written G Suite by the way

Thanks! I'll fix that

Reply:
I gave it more thought and I guess "Alternate" still makes sense here; it's an alternate to the email address you're buying.